src/libraries/System.Private.CoreLib/src/System/Runtime/CompilerServices/ValidateMethodImplRemainsInEffectAttribute.cs

```diff
@@ -0,0 +1,11 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Runtime.CompilerServices
+{
+    [AttributeUsage(AttributeTargets.Class, AllowMultiple = false, Inherited = false)]
+    public sealed class ValidateMethodImplRemainsInEffectAttribute : Attribute
```

Comment:
This should be run through API review.

Also, should the attribute be used in any of the tests?

Reply:
Who should I talk to for the API review?

> Also, should the attribute be used in any of the tests?

Yes, there are some tests with invalid return types that expect the presence of this attribute to cause a TypeLoadException to be thrown.

Reply:
> Who should I talk to for the API review?

Look for recent email from Immo titled "API Review"

Reply:
https://github.com/dotnet/runtime/issues/35315